MavenParent/pom.xml

```diff
@@ -19,11 +19,11 @@
         <license.regex.CONFIDENTIAL><![CDATA[\A/\*$\n^ \* Copyright .?. \d\d\d\d, [^$]*$\n \* All rights reserved.$\n^ \*/$\n^]]></license.regex.CONFIDENTIAL>
         <license.regex.MIT>\A/\*$\n^ \* The MIT License$$\n^ \* Copyright .?. \d\d\d\d [^$]*$\n^ \*$\n^ \* Permission is hereby granted, free of charge, to any person obtaining a copy$\n^ \* of this software and associated documentation files \(the "Software"\), to deal$\n^ \* in the Software without restriction, including without limitation the rights$\n^ \* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell$\n^ \* copies of the Software, and to permit persons to whom the Software is$\n^ \* furnished to do so, subject to the following conditions\:$\n^ \*$\n^ \* The above copyright notice and this permission notice shall be included in$\n^ \* all copies or substantial portions of the Software.$\n^ \*$\n^ \* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR$\n^ \* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,$\n^ \* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE$\n^ \* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER$\n^ \* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,$\n^ \* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN$\n^ \* THE SOFTWARE.$\n^ \*/$\n^</license.regex.MIT>
-        <maven.analysis.configuration>1.0.16-SNAPSHOT</maven.analysis.configuration>
+        <maven.analysis.configuration>${project.parent.version}</maven.analysis.configuration>
```

Comment:
This property gets overridden in down stream consuming projects.   Their parent version is what this version would be evaluated to.....   obviously not what we want.   I'll add the other fix in though.